### DISCUSSION TITLE
Replace `reqwest` with `http`

### DISCUSSION THREAD

**Discussion by okhuz** [XAMPPRocky/octocrab#168] [Q&A] [answered]
For [this](https://github.com/XAMPPRocky/octocrab/issues/99) issue, what is the main reason to start this change?

**Comment by XAMPPRocky** [comment, marked as answer]
The issue is pretty self explanatory. I want octocrab to be client agnostic, and not depend on reqwest. It only depends on reqwest for historical reasons, as the tower traits didn’t really exist to the same level when I originally made it.
